inc/kmer_unordered_set.hpp

```diff
@@ -22,19 +26,30 @@ class KmerSets {
 		// path to kmer parent files as input from command line
 		path hap1_kmer_fa_path;
 		path hap2_kmer_fa_path;
-
-		int hap1_kmer_count;
-		int hap2_kmer_count;
+		float num_hap1_kmers;
+		float num_hap2_kmers;
+		// component and haplotype 
+		int graph_component;
+		int component_haplotype;
+		static const int parent_hap1_int = 0; // hg03 paternal 
+		static const int parent_hap2_int = 1; // hg04 maternal
+		// < component,  [component_hap_path][parent_hap_int] >
+		map<int, float [2][2]> component_map;
```

Comment:
Can we use the std::array object to keep things consistent?

Reply:
This is an array, right? https://www.cplusplus.com/doc/tutorial/arrays/#:~:text=Edit%20%26%20Run-,Multidimensional,-arrays

Reply:
Yea totally, but if you use the modern c++ object from <array> then you can use consistent helper functions (like 'size()', 'at()') as with all the other containers that are indexable, and use the same syntax to choose whether it is passed by value/ref/pointer